changelog/seqera-enterprise/v24.2_major.mdx

```diff
@@ -108,11 +108,39 @@ Seqera Enterprise version 24.2 introduces a number of new features to Data Studi
 
 ## Breaking changes and warnings
 
+### Redisson properties deprecated
+
 From this version of Seqera Platform, `redisson.*` properties will be deprecated. If you have set `redisson.*` properties directly, do the following:
 
 - Replace `/redisson/*` references for the AWS parameter store with `TOWER_REDIS_*`.
 - Replace `redisson.*` references for the `tower.yml` with `TOWER_REDIS_*`. You can set `TOWER_REDIS_*` values directly in the YAML file (for example, `TOWER_REDIS_URL: redis://...`) in `tower.yml`.
 
+### MariaDB driver: New MySQL connection parameter required
```

Comment:
Is this a new breaking change introduced in v24.2, or the same breaking change from v24.1?

Reply:
From 24.1.